pyscriptjs/tests/unit/pyconfig.test.ts

```diff
@@ -1,23 +1,67 @@
-import type { AppConfig, RuntimeConfig } from '../../src/runtime';
 import { PyConfig } from '../../src/components/pyconfig';
+// eslint-disable-next-line
+// @ts-ignore
+import covfefeConfig from './covfefe.json';
+import {jest} from '@jest/globals';
 
 customElements.define('py-config', PyConfig);
 
 describe('PyConfig', () => {
     let instance: PyConfig;
+
+    const xhrMockClass = () => ({
+        open            : jest.fn(),
+        send            : jest.fn(),
+        responseText    : JSON.stringify(covfefeConfig)
+    });
+    // @ts-ignore
+    window.XMLHttpRequest = jest.fn().mockImplementation(xhrMockClass)
+
     beforeEach(() => {
         instance = new PyConfig();
-        let runtime_config: RuntimeConfig = {src: "/demo/covfefe.js", name: "covfefe", lang: "covfefe"};
-        let app_config: AppConfig = {autoclose_loader: true, runtimes: [runtime_config]};
-        instance.values = app_config;
     });
 
     it('should get the Config to just instantiate', async () => {
         expect(instance).toBeInstanceOf(PyConfig);
     });
 
     it('should load runtime from config and set as script src', () => {
+        instance.values = covfefeConfig;
         instance.loadRuntimes();
         expect(document.scripts[0].src).toBe("http://localhost/demo/covfefe.js");
     });
+
+    it('should load the default config from pyscript.json', ()=> {
+        instance.connectedCallback();
+        // @ts-ignore
+        expect(instance.values.runtimes[0].lang).toBe("python");
```

Comment:
better to load the default config and look through the keys, no?

Reply:
yeah so, everywhere in this test we use `covfefe`, not supplying anything means it will load the default one i.e. `python` one -- but I have now additionally checked for `author email` and `name` also.